src/Basket.js

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
 
-const Basket = () => {
+const Basket = (props) => {  
   return (
     <section>
       <p>Shopping Cart</p>
       <ul>
-        <li>Phone 1 <button>x</button></li>
-        <li>Phone 2 <button>x</button></li>
-        <li>Phone 3 <button>x</button></li>
+      {Object.keys(props.basketItems).map((phone) => (
+        <li key={phone}>{phone} {(props.basketItems[phone] > 1) ? '(' + props.basketItems[phone] + ')' : null}<button onClick={()=> {
```

Comment:
First, use destructuring to get rid of repeated `props.` in the code.
```javascript
const {basketItems, handleRemoveFromBasket} = props;
```
Then, you remember we used `Object.entries` so we have variable `count` now.
```suggestion   
        <li key={phone}>{phone}  {(count > 1) ? `(${count})` : ''}<button onClick={()=> {
```